docs/datagrid/understanding-dataGrid.md

```diff
@@ -0,0 +1,96 @@
+Understanding DataGrid
+
+===
+
+The DataGrid is a tool that can help you to vizualize your data in a table.
+
+![datagrid.png](datagrid.png)
+
+Defaults
+---------
+Default options for the all rows or all columns.
+
+```
+    defaults: {
+        columns: {
+            editable: true
+        }
+    }
+```
+
+For more information on defaults options see the [API reference]().
+
+Caption
+---------
+
+The caption of the datagrid grid.
+
+```
+    caption: {
+        text: 'Your title of datagrid'
+    }
+```
+
+For more information on caption options see the [API reference]().
+
+Resizer
+---------
+
+Allows you to resize the entire column. The functionality is enabled by default,
+but you can disable it in the settings option.
+
+```
+    settings: {
+        columns: {
+            resizing: false
+        }
+    }
+```
+
+For more information on resizer options see the [API reference]().
+
+Cell
+---------
+
+The basic element on the DataGrid, can be formatted by `cellFormat` or `cellFormatter`.
+You can also set the `useHTML` option and apply the custom HTML in formatters.
+
+```
+    defaults: {
+        columns: {
+            useHTML: true
+        }
+    },
+    columns: {
+        column1: {
+            cellFormat: '<h3>{value}</h3>'
+        },
+        column2: {
+            cellFormatter: function () {
+                return '<a href="' + this.value + '">URL</a>';
+            }
+        }
+    }
+```
+
+For more information on cell options see the [API reference]().
+
+Editor
```

Comment:
Do we want to call it `editor`?

Reply:
I was hesitating, my first thought was to call it `Cell Editor`, but it sounds a bit strange as well.

I'm open to suggestions.

Reply:
Cell Value Editor? Value Editor?

Reply:
Value editor sounds good